hack/community.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Include [WINC-700](https://issues.redhat.com/browse/WINC-700) as a prefix in the PR title.